src/main/scala/TraceIO.scala

```diff
@@ -77,39 +122,77 @@ class TileTraceIO(val insnWidths: TracedInstructionWidths, val numInsns: Int) ex
 }
 
 // The IO matched on by the TracerV bridge: a wrapper around a heterogenous
-// bag of vectors. Each entry is Vec of committed instructions
+// bag of vectors. Each entry is trace associated with a single tile (vector of committed instructions + clock + reset)
 class TraceOutputTop(val widths: Seq[TracedInstructionWidths], val vecSizes: Seq[Int]) extends Bundle {
   val traces = Output(HeterogeneousBag(widths.zip(vecSizes).map({ case (w, n) => new TileTraceIO(w,n) })))
 }
 
 object TraceOutputTop {
-  def apply(proto: Seq[Vec[TracedInstruction]]): TraceOutputTop =
-    new TraceOutputTop(proto.map(t => TracedInstructionWidths(t.head)), proto.map(_.size))
+  def apply(proto: Seq[Vec[TracedInstruction]], protoExt: Seq[Vec[ExtendedTracedInstruction]]): TraceOutputTop =
```

Comment:
i guess it doesn't cost anything to do it this way but i'm curious why you choose not to simply plumb out a single IO with all possible signals. The bridge/BB specific bundle could have a constructor which accepted that extended type and only connected the pertinent signals.

Reply:
IIRC (since it has been a long time), RC only gives out a `Vec[TracedInstructions]` while BOOM now gives out `Vec[ExtTracedInstructions]`. So to make it easy on me, I just passed in both directly from the traceNexus'es instead of trying to convert the `TracedInstructions` from RC to an extended type. In retrospect, that might be a bit cleaner, I think I can change that in a separate PR just to let this one go in.

Reply:
I think this decision was motivated by me trying to avoid changing RC.